frontend/src/bundles/common/components/slider/styles.ts

```diff
@@ -0,0 +1,45 @@
+const defaultSliderStyle = {
+    height: 15,
+    marginTop: '20px',
+    '& .MuiSlider-markLabel': {
+        transform: 'translateX(0%)',
+        '&:nth-of-type(n + 5)': {
+            display: 'none',
+        },
+    },
+    '& .MuiSlider-track': {
+        borderTopRightRadius: 0,
+        borderBottomRightRadius: 0,
+    },
+    '& .MuiSlider-thumb': {
+        width: '1px',
+        height: '15px',
+        backgroundColor: '#000',
+        '&:focus, &:hover, &.Mui-active, &.Mui-focusVisible': {
+            boxShadow: 'inherit',
+        },
+        '&:before': {
+            top: '-5px',
+            width: '0px',
+            height: '0px',
+            borderStyle: 'solid',
+            borderWidth: '5px 5px 0 5px',
+            borderColor: '#000000 transparent transparent transparent',
+            transform: 'rotate(0deg)',
+        },
+    },
+    '& .MuiSlider-valueLabel': {
+        padding: 0,
+        top: '-5px',
+        backgroundColor: '#fff',
+        color: 'initial',
+        '&:before': {
+            display: 'none',
+        },
+    },
+    '& .MuiSlider-mark': {
+        opacity: 0,
+    },
+};
+
+export { defaultSliderStyle };
```

Comment:
Move it to styles.module.scss and using module styles

Reply:
what is the level of nesting? can I make it like that:
```scss
    /* parent and inside: */
    &.MuiSlider-markLabel {
        transform: 'translateX(0%)';
        &:nth-of-type(n + 5) {
            display: none;
        }
    }
```

Reply:
@JemiJe check this comment please - https://github.com/BinaryStudioAcademy/bsa-2023-bsa-talents/pull/77#discussion_r1303459256